src/screens/TopicPageScreen/index.js

```diff
@@ -15,6 +15,7 @@ import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
 import dimen from '../../utils/dimen';
 import removePrefixTopic from '../../utils/topics/removePrefixTopic';
 import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useViewPostTimeHook` is correctly added. This hook is necessary for the new functionality.
```diff
+ import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed